doc/src/conf.py

```diff
@@ -32,7 +32,7 @@
 mathjax_path = 'http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML-full'
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['.templates']
+templates_path = ['_templates']
```

Comment:
Calling it `_templates` seems to be pretty common too: https://github.com/search?utf8=%E2%9C%93&q=%22templates_path%22+conf.py&type=Code